client/my-sites/site-monitoring/main.tsx

```diff
@@ -19,14 +16,7 @@ interface SiteMetricsProps {
 }
 
 export function SiteMetrics( { tab = 'metrics' }: SiteMetricsProps ) {
-	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
-	const isGlobalSiteViewEnabled = useSelector( ( state ) =>
-		getIsGlobalSiteViewEnabled( state, siteId )
-	);
-
-	const titleHeader = isGlobalSiteViewEnabled
-		? translate( 'Monitoring' )
-		: translate( 'Site Monitoring' );
```

Comment:
After `layout/dotcom-nav-redesign-v2` is dropped, this is a dead code now as this was meant to be shown when Site Monitoring is still in "Calypso" (not in the hosting dashboard).